src/SDL2.cs

```diff
@@ -2544,12 +2662,31 @@ public struct SDL_Color
 		}
 
 		[StructLayout(LayoutKind.Sequential)]
+		public unsafe struct SDL_ColorArray
```

Comment:
Again, this is necessary to make this functionality available to implementation languages that don't have `fixed`.  Feel free to suggest a better solution if you can think of one.